### ISSUE TITLE
Video file not found in assets folder

### ISSUE THREAD

**Issue by MiSu2002** [koolkishan/netflix-clone-react-node#3]
After running the project, an error encountered that video.mp4 is not present in assets folder
<img width="1448" alt="Screenshot 2023-01-17 at 12 46 06 PM" src="https://user-images.githubusercontent.com/89409581/212833672-18fc2cb7-2d0d-426e-83e9-c7531e1e1817.png">

**Comment by dixit18** [comment]
i'm getting same problem

**Comment by sachin-vinod** [comment]
This problem you guys getting because when we push play button which is on the netflix or we can say ("/") component, in code in assets folder there is no video, so you can put one by own, author forget to add video in the assets

**Comment by MiSu2002** [comment]
@sachin-vinod You are right... It's just a small issue that can be easily fixed. But it really is not necessary to play the same video for all movies. I hope the author updates the code for better UX.

**Comment by debsouryadatta** [comment]
Is there any way to have different videos in different movie hovers, not manually!

**Comment by MiSu2002** [comment]
@debsouryadatta You can use API to fetch movies or tv video. If you are using TMDB API, it's available there, check the TMDB API documentation. You can fetch using the movie or tv id.

**Comment by debsouryadatta** [comment]
@MiSu2002 ok cool, I will go through it
